src/MeetingsSDKAdapter.js

```diff
@@ -561,6 +563,7 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
         },
         localShare: {
           stream: null,
+          ...(!isGetDisplayMedia && {error: 'Share screen is not supported'}),
```

Comment:
this condition is not dependent on the meeting, it is dependent on the browser.

we can implement it just in the control.

```js

const browserSupport = ...;

return !broserSupport ? of(...not-supported-disabled-state..) : this.adapter.getMeeting(...)etc;